packages/manager/apps/pci-public-ip/src/pages/order/steps/FloatingSteps.tsx

```diff
@@ -64,6 +75,9 @@ export const FloatingSteps = ({
     }
   }, [selectedInstanceIpAddresses]);
 
+  const has3AZ = isRegionWith3AZ(DataState.regions);
+  const pciCommonProperties = usePCICommonContextFactory({ has3AZ });
```

Comment:
Name should be agnostic from library :)

As an example, if you are using tanstack query provider, you will not give as value something like `const tanstakQueryContext` ;)